src/paths.rs

```diff
@@ -1,8 +1,15 @@
+/// Combine two paths into a single path, possibly inserting a '/' between them.
 pub fn make_path(parent: &str, child: &str) -> String {
-    let mut result = parent.to_owned();
-    if parent.chars().last().unwrap() != '/' {
-        result.push('/');
+    if parent.chars().last() == Some('/') {
```

Comment:
I did a micro-benchmark of this...`format!` appears to be faster than `push`+`push_str`.